pkg/k8s/aggregator.go

```diff
@@ -81,6 +81,17 @@ func (i *IngressAggregator) List() ([]v1beta1.Ingress, error) {
 			if !ok {
 				return nil, fmt.Errorf("unexpected object in store: %+v", obj)
 			}
+
+			if len(ingress.Status.LoadBalancer.Ingress) < 1 {
+				return nil, fmt.Errorf("no ingress load balancer in store: %+v", obj)
+			}
+
```

Comment:
I'm not sure we want to error here, this means that if you have any ingress in your cluster that's missing these fields yggdrasil would be unable to generate envoy config.